pkg/repos/get.go

```diff
@@ -112,11 +113,29 @@ func (m *Manager) deferredSetUpCredentialHelpers(ctx context.Context, cliCfg *co
 	locker.Lock("gptscript-credential-helpers")
 	defer locker.Unlock("gptscript-credential-helpers")
 
+	// Load the last-checked file to make sure we haven't checked the repo in the last 24 hours.
+	lastChecked, err := os.ReadFile(m.credHelperDirs.LastCheckedFile)
+	if err == nil {
+		if t, err := time.Parse(time.RFC3339, strings.TrimSpace(string(lastChecked))); err == nil && time.Since(t) < 24*time.Hour {
+			// Make sure the binary still exists, and if it does, return.
+			if _, err := os.Stat(filepath.Join(m.credHelperDirs.BinDir, "gptscript-credential-"+helperName+suffix)); err == nil {
+				log.Debugf("Not checking for new version of credential helper %s, last checked %v", helperName, t)
+				return nil
+			}
+		}
+	}
+
+	// Load the credential helpers repo information.
 	_, repo, _, err := github.Load(ctx, nil, credentialHelpersRepo)
 	if err != nil {
 		return err
 	}
 
+	// Update the last-checked file.
+	if err := os.WriteFile(m.credHelperDirs.LastCheckedFile, []byte(time.Now().Format(time.RFC3339)), 0644); err != nil {
```

Comment:
super-nit: If I wanted to be really pedantic, I'd recommend storing `time.Now()` in a variable, when you declare `lastChecked` above, and base all the comparisons off of that.

Reply:
Fair enough lol